packages/menu/README.md

```diff
@@ -70,11 +71,17 @@ const Menu = () => {
         <>
           <button {...getTriggerProps()}>Menu</button>
           <ul {...getMenuProps()} style={{ visibility: isExpanded ? 'visible' : 'hidden' }}>
-            {items.map(item => (
-              <li key={item.value} {...getItemProps({ item })}>
-                {item.label}
-              </li>
-            ))}
+            {items.map(item =>
+              item.href ? (
+                <li key={item.value} role="none">
+                  <a {...getItemProps({ item })}>{item.label}</a>
+                </li>
+              ) : (
+                <li key={item.value} {...getItemProps({ item })}>
+                  {item.label}
+                </li>
+              )
+            )}
```

Comment:
Anchor items are pretty common in menus, so it felt prudent to give a first class example.